### ISSUE TITLE
Editing an event that is defined without a date shows date field twice - demo-dev

### ISSUE THREAD

**Issue by melizaa** [fisharebest/webtrees#1685]
When I edit an individual- or family-event that exists in the Gedcom without a date, I see Date 2 times in the edit form.

If I edit an event that has a date or I add a new event, I see the date as I should, only once.

See the Not married fact of f47 in demo-dev.
![2018-04-11_00-31-58](https://user-images.githubusercontent.com/5202996/38584716-d1721f9c-3d1f-11e8-8124-9b0bdc426537.jpg)

**Comment by fisharebest** [comment]
We have two pieces of logic that add missing subtags to facts/events.

The first is adding the `DATE`, because the fact is not listed as a "no date fact".

The second adds a default set of subtags to custom fact/events, which includes `DATE`.

This is a custom tag (begins with `_`), but we use it as a standard tag, and it gets handled by both pieces of logic.

**Comment by fisharebest** [comment]
This fix has been merged into the main branch, and will be included in 2.1.0.
